package.json

```diff
@@ -48,7 +48,7 @@
     "keycharm": "^0.2.0",
     "moment": "^2.24.0",
     "timsort": "^0.3.0",
-    "vis-data": "^6.1.1",
+    "vis-data": "^6.2.0",
```

Comment:
This could be a big change and should be done in a seperate PR, please.

Reply:
@mojoaxel if you checkout the repo and look at dist/vis-network.js it contains
`* vis-data - data
   * http://visjs.org/
   * 
   * Manage unstructured data using DataSet. Add, update, and remove data, and listen for changes in the data.
   * 
   * @version 6.2.0
   * @date    2019-08-14T22:18:56Z`

So it was clear to me to update the dependencies.  
Looks like someone did it already but didn't commit it in repo.

Must I revert this line for this request?

Reply:
We have a auto-release feature. That is we each change should a seperate PR. I think it would be the best if you would change the whole package.json back to its original. Than we can do the dependency update later in a seperate PR. Sorry for the inconvenience!

Reply:
Just a note here: The files on npm are regenerated according to package lock on each release, that is npm doesn't have 6.2.0 yet. Dist is updated manually and someone (that would be me as the last commit to dist is mine) messed up (probably didn't reinstall all packages after switching branches or didn't update dist after reverting dependency update) and pushed wrong version of dist to GitHub.

Reply:
@mojoaxel I turned back package.json and package-lock.json r3code@2833cf2

Reply:
@Thomaash Do I need to propose PR to update dependencies?

Reply:
If you want to update it now then yes but we'll update it ourselves eventually.